src/instructlab/server.py

```diff
@@ -155,12 +171,23 @@ def server(
             return
         raise ServerException(f"failed creating the server application: {exc}") from exc
 
+    template = ""
+    eos_token = "<|endoftext|>"
+    bos_token = ""
+    for template_type in templates:
+        if template_type["model"] == model_family:
+            template = template_type["template"]
+            if template == "mixtral":
+                eos_token = "</s>"
+                bos_token = "<s>"
```

Comment:
```suggestion
    for template_dict in templates:
        if template_dict["model"] == model_family:
            template = template_dict["template"]
            if template_dict["model"] == "mixtral":
                eos_token = "</s>"
                bos_token = "<s>"
```

Reply:
Thanks. I caught this as well and will be fixing in my next push